### PR TITLE
LPS-131199 | Add Categories#AddCategoryProperties to AUI modules relevant testing

### PR DESCRIPTION
**Pull request by john-co**
**Description**
Include a test to cover FP4 regression https://issues.liferay.com/browse/LPS-130905

**Test Plan**
- [x] relevant test includes Categories#AddCategoryProperties
- [x] Categories#AddCategoryProperties passes

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-131199

### CODE REVIEW

File: modules/apps/frontend-js/test.properties [hunk 1]
```diff
@@ -14,9 +14,14 @@
...
 
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
-        (environment.acceptance == true) AND \
-        (portal.acceptance != quarantine) AND \
-        (portal.upstream != quarantine) AND \
+        (
+            (environment.acceptance == true) OR \
+            (testray.component.names ~ "AUI")
+        ) AND \
+        (
```
Comment (kenjiheigel) [inline modules/apps/frontend-js/test.properties:21]: @john-co , I believe the syntax is wrong. For lines, 17, 20, 21, there should be a `\` at the end of the line. Could you try that and run again?

Comment (john-co) [reply modules/apps/frontend-js/test.properties]: Ah, thanks @kenjiheigel!

**Comment by kenjiheigel** [comment]
ci:test:relevant

**Comment by john-co** [comment]
- Test is included in [ci:test:relevant and is passing.](https://testray.liferay.com/home/-/testray/case_results/1562987347?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayCaseName%3DAddCategoryProperties%26testrayBuildId%3D1562986743%26testrayRunId%3D1562987322%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)

- other test failures look unrelated.

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**